app/assets/styles/styles.css

```diff
@@ -33,194 +31,120 @@ p {
   line-height: 1.55;
 }
 
-.newsletter-date {
-  color: #fdfd56;
-  font-size: 0.8rem;
+.logo__btn {
+  display: flex;
+  justify-content: center;
+  margin-top: 0.7em;
+  margin-bottom: 0.7em;
 }
 
-.language-menu {
-  width: 100%;
-  font-size: 1.15rem;
-  font-family: 'Trajan Pro Regular', 'Times New Roman', serif;
-  text-align: right;
-  background: #1a1451;
-  margin-bottom: 0.625em;
-  padding: 0.5em 1em;
+@media only screen and (min-width: 400px) {
+  .logo__btn {
+    display: block;
+    position: absolute;
+    top: 0;
+    margin-top: 0;
+    margin-left: 1.5em;
+  }
 }
 
-.language-menu--item {
-  color: #fff;
-  padding: 0.475em;
+.menu {
+  display: flex;
+  justify-content: space-between;
+  font-size: 0.95rem;
+  font-family: 'Trajan Pro Bold', 'Times New Roman', serif;
+  background: #1a1451;
+  padding: 0.5em 1.5em;
 }
 
-.language-menu--item:hover {
+.menu__news-name {
   color: #fdfd56;
+  letter-spacing: 0.19em;
```

Comment:
We just want  a touch of letter spacing `0.01em`. Visually, the letters look too far apart, it starts to look like letters instead of one word. The idea is to give the letters a bit of breathing room while still maintaining the visual appearance of being one word.